packages/javascript/bh-shared-ui/src/views/PrivilegeZones/Save/TagForm/TagForm.tsx

```diff
@@ -510,7 +531,23 @@ export const TagForm: FC = () => {
                                             name='require_certify'
                                             render={({ field }) => (
                                                 <FormItem>
-                                                    <FormLabel>Require Certification</FormLabel>
+                                                    <FormLabel className='flex gap-2 items-center'>
+                                                        Require Certification
```

Comment:
I wonder if this should be swapped to `Enable Certification` to be consistent with the other switches?

That said, the tooltip regression also removed the description field for this switch, can you please give this the `enabled | disabled` treatment as well?